modules.d/01systemd-resolved/module-setup.sh

```diff
@@ -36,7 +36,6 @@ install() {
     inst_multiple -o \
         "$dbussystem"/org.freedesktop.resolve1.conf \
         "$dbussystemservices"/org.freedesktop.resolve1.service \
-        "$systemdutildir"/resolv.conf \
```

Comment:
`/usr/lib/systemd/resolv.conf` exists in Fedora, I am not sure if we should remove this file from `systemd-resolved`

Reply:
I see. Let me close this PR than. Thanks @LaszloGombos